src/admin/paths/admin@expenses.yaml

```diff
@@ -23,6 +23,12 @@ get:
                 type: array
                 items:
                   $ref: '../../components/schemas/expense.yaml#/expense_out'
```

Comment:
Created task to add 400 bad request as one of the response type to all GET apis once this is merged in. https://app.clickup.com/t/2at0ztt